### PR TITLE
Meth type obfuscation

### PR DESCRIPTION
**Pull request by kkosowski-bt**

### CODE REVIEW

File: MachObfuscator/Options/Options.swift [hunk 1]
```diff
@@ -44,23 +52,33 @@ extension Options {
...
+                  methTypeObfuscation: methTypeObfuscation,
+                  machOViewDoom: machOViewDoom,
+                  manglerType: manglerType,
+                  appDirectory: appDirectoryURL)
     }
 
     static var usage: String {
         return """
-        usage: \(CommandLine.arguments[0]) [-qvh] [-m mangler_key] APP_BUNDLE
+        usage: \(CommandLine.arguments[0]) [-qvhD] [-m mangler_key] APP_BUNDLE
```
Comment (kam800) [inline MachObfuscator/Options/Options.swift]:
```suggestion
        usage: \(CommandLine.arguments[0]) [-qvhtD] [-m mangler_key] APP_BUNDLE
```

File: MachObfuscator/Options/Options.swift [hunk 2]
```diff
@@ -44,23 +52,33 @@ extension Options {
...
 
           Obfuscates application APP_BUNDLE in-place.
 
         Options:
           -h              help screen (this screen)
           -q              quiet mode, no output to stdout
           -v              verbose mode, output verbose info to stdout
+          -t              obfuscate methType section (may cause runtime problems)
```
Comment (kam800) [inline MachObfuscator/Options/Options.swift]:
```suggestion
          -t              obfuscate methType section (objc/runtime.h methods may work incorrectly)
```

Comment (kkosowski-bt) [reply MachObfuscator/Options/Options.swift]: awesome suggestion, thx!

File: MachObfuscator/Mach/Mach+Parsing.swift [hunk 3]
```diff
@@ -15,6 +15,13 @@ extension Mach {
         return classNamesData.split(separator: 0).compactMap { String(bytes: $0, encoding: .utf8) }
     }
 
+    var methTypes: [String] {
```
Comment (kam800) [inline MachObfuscator/Mach/Mach+Parsing.swift]: Is it still needed?

File: MachObfuscator/SymbolsCollecting/ObfuscationSymbols.swift [hunk 4]
```diff
@@ -9,4 +9,5 @@ struct ObfuscationSymbols {
 struct ObjCSymbols {
     var selectors: Set<String>
     var classes: Set<String>
+    var methTypes: Set<String>
```
Comment (kam800) [inline MachObfuscator/SymbolsCollecting/ObfuscationSymbols.swift]: Is it still needed?

File: MachObfuscator/SymbolsCollecting/SymbolsSource.swift [hunk 5]
```diff
@@ -1,6 +1,7 @@
 protocol SymbolsSource {
     var selectors: [String] { get }
     var classNames: [String] { get }
+    var methTypes: [String] { get }
```
Comment (kam800) [inline MachObfuscator/SymbolsCollecting/SymbolsSource.swift]: Is it still needed?

**Pull request merged**